spec/jobs/reports/identity_verification_report_spec.rb

```diff
@@ -1,21 +1,66 @@
 require 'rails_helper'
 
 RSpec.describe Reports::IdentityVerificationReport do
+  let(:report_date) { Date.new(2023, 12, 12).in_time_zone('UTC') }
+  let(:team_ada_email) { 'ada@example.com' }
+
   before do
     allow(IdentityConfig.store).to receive(:s3_reports_enabled).and_return(true)
+    allow(IdentityConfig.store).to receive(:team_ada_email).and_return(team_ada_email)
```

Comment:
If you're not using `team_ada_email` for the expectation on line 56, then you can delete line 5 and just put in the email address here.
```suggestion
    allow(IdentityConfig.store).to receive(:team_ada_email).and_return('ada@example.com')
```